examples/create_identity_provider/github/README.md

```diff
@@ -0,0 +1,60 @@
+# GitHub identity provider example
+
+Configuring [GitHub authentication](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/authorizing-oauth-apps) allows users to log in to OpenShift Container Platform with their GitHub credentials.
+To prevent anyone with any GitHub user ID from logging in to your OpenShift Container Platform cluster, you can restrict access to only those in specific GitHub organizations.
+
+## Prerequisites
+
+### An OpenShift cluster
+
+This example assumes you have created a cluster via OCM, It can be via this terraform provider or by a different client.
+We will need the cluster name and some credentials.
+
+### Setting up your application in GitHub
+
+To use GitHub or GitHub Enterprise as an identity provider, you must register an application to use.
+
+Procedure
+
+1. Register an application on GitHub:
+    
+    - For GitHub, click [**Settings**](https://github.com/settings/profile) → [**Developer settings**](https://github.com/settings/apps) → [**OAuth Apps**](https://github.com/settings/developers) → [**Register a new OAuth application**](https://github.com/settings/applications/new).
+        
+    - For GitHub Enterprise, go to your GitHub Enterprise home page and then click **Settings → Developer settings → Register a new application**.
+        
+    
+2. Enter an application name, for example `My OpenShift Install`.
+    
+3. Enter a homepage URL, such as `https://oauth-openshift.apps.<cluster-name>.<cluster-domain>`.
+    
+4. Optional: Enter an application description.
+    
+5. Enter the authorization callback URL, where the end of the URL contains the identity provider `name`:
+    
+    `https://oauth-openshift.apps.<cluster-name>.<cluster-domain>/oauth2callback/<idp-provider-name>`
+    
+    For example:
+    `https://oauth-openshift.apps.openshift-cluster.example.com/oauth2callback/Github`
+	
+	> **Note**: `<idp-provider-name>` is case-sensitive.
```

Comment:
Add here link to the place in the example which must be identical
In the example you used GitHUb